content/actions/creating-actions/dockerfile-support-for-github-actions.md

```
@@ -83,9 +83,10 @@ Using the example Dockerfile above, {% data variables.product.product_name %} wi
 ``` sh
 #!/bin/sh
 
-# `$*` expands the `args` supplied in an `array` individually
-# or splits `args` in a string separated by whitespace.
-sh -c "echo $*"
+# `$#` expands to the number of arguments and `$@` expands to the supplied `args`
+printf '%d args:' "$#"
```

Putting quotes around each arg, and removing the blank echo at the end.

```suggestion
printf '%d args:' "$#"
printf $' \'%s\'' "$@"
```

---

The blank echo is used to print a newline after the arguments. Could also use `printf '\n'`.

It looks like what was merged is a mix of both now. I don't mind using single quotes around the arguments but `$''` is not POSIX. Could use `printf " '%s'" "$@"` instead.

---

Thanks a lot, @soliton-. I also applied my change to the wrong lines 🤦‍♂️. Fixed in this PR: https://github.com/github/docs/pull/22635